decko-rails/lib/decko/engine.rb

```diff
@@ -27,7 +26,7 @@ def gem_root
 
   end
 
-  class Decko::Engine < Rails::Engine
+  class Engine < ::Rails::Engine
     
     paths.add "app/controllers", :eager_load => true
     paths.add 'gem-assets',      :with => 'public/assets'
```

Comment:
note: when we inherit from this, we won't be able to add these paths like this any more!